gql/transport/requests.py

```diff
@@ -1,26 +1,66 @@
 from __future__ import absolute_import
 
+from typing import Any, Dict, Union
+
 import requests
 from graphql.execution import ExecutionResult
+from graphql.language.ast import Node
 from graphql.language.printer import print_ast
+from requests.auth import AuthBase
+from requests.cookies import RequestsCookieJar
 
-from .http import HTTPTransport
+from gql.transport.transport import Transport
 
 
-class RequestsHTTPTransport(HTTPTransport):
-    def __init__(self, url, auth=None, use_json=False, timeout=None, **kwargs):
-        """
-        :param url: The GraphQL URL
-        :param auth: Auth tuple or callable to enable Basic/Digest/Custom HTTP Auth
-        :param use_json: Send request body as JSON instead of form-urlencoded
-        :param timeout: Specifies a default timeout for requests (Default: None)
+class RequestsHTTPTransport(Transport):
+    def __init__(
+        self,  # type: RequestsHTTPTransport
+        url,  # type: str
+        headers=None,  # type: Dict[str, Any]
+        cookies=None,  # type: Union[Dict[str, Any], RequestsCookieJar]
+        auth=None,  # type: AuthBase
+        use_json=False,  # type: bool
+        timeout=None,  # type: int
+        verify=True,  # type: bool
+        **kwargs  # type: Any
+    ):
+        """Create a HTTPTransport using requests library to execute GraphQL queries on remote servers.
```

Comment:
Suggestion:

class docstring:

    """Transport to execute GraphQL queries on remote servers. 

    The transport uses the requests library to send HTTP POST request.
    """

init docstring:

    """Initialize the transport with the given request parameters.

    :prarm url: ... etc
    """